### PR TITLE
change daemon and mac dir

### PR DESCRIPTION
**Pull request by Pega88**
@ch4ot1c daemon rename & mac folder rename for blockchain & wallet.
will create issue for linux/win since don't know exact folders

### CODE REVIEW

File: src/java/com/vaklinov/zcashui/OSUtil.java [hunk 1]
```diff
@@ -218,12 +218,14 @@ public static String getBlockchainDirectory()
 		
 		if (os == OS_TYPE.MAC_OS)
 		{
-			return new File(System.getProperty("user.home") + "/Library/Application Support/Zen").getCanonicalPath();
+			return new File(System.getProperty("user.home") + "/Library/Application Support/Zlassic").getCanonicalPath();
```
Comment (ch4ot1c) [inline src/java/com/vaklinov/zcashui/OSUtil.java]: typo!

File: src/java/com/vaklinov/zcashui/OSUtil.java [hunk 2]
```diff
@@ -218,12 +218,14 @@ public static String getBlockchainDirectory()
...
 		} else if (os == OS_TYPE.WINDOWS)
 		{
+			//todo ch4ot1c?
 			return new File(System.getenv("APPDATA") + "\\Zen").getCanonicalPath();
```
Comment (ch4ot1c) [inline src/java/com/vaklinov/zcashui/OSUtil.java]: APPDATA resolves to C:\Users\You\AppData\Roaming\

File: src/java/com/vaklinov/zcashui/OSUtil.java [hunk 3]
```diff
@@ -218,12 +218,14 @@ public static String getBlockchainDirectory()
...
 		} else
 		{
+			//todo ch4ot1c?
 			return new File(System.getProperty("user.home") + "/.zen").getCanonicalPath();
```
Comment (ch4ot1c) [inline src/java/com/vaklinov/zcashui/OSUtil.java]: users.home resolves to /Users/You/

File: src/java/com/vaklinov/zcashui/OSUtil.java (hunk 1 above)
Comment (Pega88) [reply src/java/com/vaklinov/zcashui/OSUtil.java]: +1

File: src/java/com/vaklinov/zcashui/OSUtil.java (hunk 2 above)
Comment (Pega88) [reply src/java/com/vaklinov/zcashui/OSUtil.java]: I mean which folder to use, \\Zen --> \\Zclassic?

File: src/java/com/vaklinov/zcashui/OSUtil.java (hunk 3 above)
Comment (Pega88) [reply src/java/com/vaklinov/zcashui/OSUtil.java]: I mean which folder to use, ./zen --> ./zclassic? it needs to be in sync with other zcl wallets, no?

File: src/java/com/vaklinov/zcashui/OSUtil.java (hunk 2 above)
Comment (ch4ot1c) [reply src/java/com/vaklinov/zcashui/OSUtil.java]: Zclassic

File: src/java/com/vaklinov/zcashui/OSUtil.java (hunk 3 above)
Comment (ch4ot1c) [reply src/java/com/vaklinov/zcashui/OSUtil.java]: zclassic

**Pull request merged**